src/include/distributed/multi_logical_optimizer.h

```diff
@@ -105,7 +105,8 @@ static const char *const AggregateNames[] = {
 	"invalid", "avg", "min", "max",
 	"sum", "count", "array_agg",
 	"jsonb_agg", "jsonb_object_agg",
-	"json_agg", "json_object_agg"
+	"json_agg", "json_object_agg",
+	"bit_and", "bit_or", "bool_and", "bool_or", "every"
 };
 
```

Comment:
Should we add these aggregates to the `AggregateType` enum to not to break the rule given in the second part of the comment above ?

Reply:
Makes sense, added them.